CMakePresets.json

```diff
@@ -15,7 +15,7 @@
       "hidden": true,
       "cacheVariables": {
         "CMAKE_BUILD_TYPE": "Debug",
-        "CMAKE_CXX_FLAGS": "-fsanitize=address -fsanitize=pointer-compare -fsanitize=pointer-subtract -fsanitize=leak -fsanitize=undefined"
+        "CMAKE_CXX_FLAGS": "-fsanitize=address -fsanitize=pointer-compare -fsanitize=pointer-subtract -fsanitize=undefined"
```

Comment:
Why remove the leak sanitizer?

Reply:
I think this is a bundled fix for #65

Reply:
`gcc` on **OSX** does not support this!

Reply:
Is this because there's a `g++` that isn't actually GCC? 
Least common denominator options is unfortunate.